### PR TITLE
feat(nest): upgrade nest to v9

### PR DESCRIPTION
**Pull request by nartc**
## Expected Behavior
support nestjs v9

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/nest/migrations.json [hunk 1]
```diff
@@ -75,6 +75,31 @@
...
+        },
+        "@nestjs/schematics": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        },
+        "@nestjs/testing": {
+          "version": "^9.0.0",
+          "alwaysAddToPackageJson": false
+        }
+      }
```
Comment (AgentEnder) [inline packages/nest/migrations.json:102]: Do we need to include @nestjs/swagger here?

Comment (nartc) [reply packages/nest/migrations.json:102]: Good catch. nestjs/swagger was indeed updated to v6. Thanks

**Review by AgentEnder — APPROVED** [review]

**Comment by samratarmas** [comment]
Hello team,

The [https://github.com/nrwl/nx/releases/tag/14.5.0](url) release note states:

Nest: upgrade nest to v9

With these references: #11120 -> [https://github.com/nrwl/nx/commit/421b4d940e4e342b1ad9be462ce2692de0f3a622](url)

However, 
On this commit: [https://github.com/nrwl/nx/commit/15880d9bd1395f85d69645db9546959eeb753903](url) ,
@AgentEnder reverted those upgrades.

As of  nx version 14.5.1 creating a new workspace and generating a nest application brings in dependencies from v8.
This means that either the release notes were not supposed to mention the v9 upgrade, or those reverted commits were not supposed to happen. Hence, it would be very helpful to know whether nx team @FrozenPandaz @nartc have plans to bring in nest v9 in current nx version 14 or are waiting for the next major nx update cycle.

Looking forward to hearing from you,
Cheers!

**Comment by nartc** [comment]
@samratarmas Sorry for the inconvenience. I'll get with @AgentEnder to see why that was the case. Thanks for reporting to us

**Pull request merged**